tools/asset-sync/README.md

```diff
@@ -0,0 +1,610 @@
+# Azure SDK Assets Relocation -- "move recordings out of repos"
```

Comment:
Did you intend to include the docx file given you are putting the content in this MD as well?

Reply:
I did. But I can also just emplace it in our sharepoint and link out to it. Given that so many folks have asked I'll just do that.